source/tutorials/apache-hadoop.rst

```diff
@@ -3,33 +3,41 @@
 Apache\* Hadoop\*
 #################
 
-This tutorial walks you through the process of installing, configuring, and
-running Apache Hadoop on |CL-ATTR|. The Apache Hadoop software library is a
-framework for distributed processing of large data sets across clusters of
-computers using simple programming models. It is designed to scale up from
-single servers to thousands of machines, with each machine offering local
-computation and storage.
+This tutorial explains the process of installing, configuring, and
+running Apache Hadoop on |CL-ATTR|.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Description
+***********
+
+The Apache Hadoop software library is a framework for distributed processing
+of large data sets across clusters of computers using simple programming
+models. It is designed to scale up from single servers to thousands of
+machines, with each machine offering local computation and storage.
 
 Prerequisites
 *************
 
-Before following this tutorial, you should follow the
+Before following this tutorial, follow the
```

Comment:
*  :ref:`bare-metal-install-desktop`  
* In |CL| run `:command: swupd update`

Reply:
Done.